wpilib/wpilib/command/command.py

```diff
@@ -49,18 +49,21 @@ def __init__(self, name=None, timeout=None):
         :param timeout: The time (in seconds) before this command "times out".
                         Default is no timeout.  See isTimedOut().
         """
+        super().__init__(False)
         self.mutex = threading.RLock()
 
         # The name of this command
         if name is None:
-            self.name = self.__class__.__name__
-        else:
-            self.name = name
+            name = self.__class__.__name__
+
+        self.setName(name)
 
         # The time (in seconds) before this command "times out" (or None if no
         # timeout)
         if timeout is not None and timeout < 0:
             raise ValueError("Timeout must not be negative")
+        if timeout is None:
+            timeout = -1.0
```

Comment:
Hmm. Is there a reason for adding this?

Reply:
yes, in the java wpilib the default value is -1.0, and other methods in this class expect it to be -1.0 and break if it is None. There are tests that exercise this.

Reply:
2 lines above you say timeout cannot be negative, yet here you set it to negative...Why not set it to -1 if it's negative?